test/nn/models/test_compile.py

```diff
@@ -47,7 +47,7 @@
                 ffn_dropout=0.5,
             ),
             None,
-            4,
+            2,
```

Comment:
Although PyTorch `2.1.1` produces 2 graph breaks, PyTorch nightly produces 4 graph breaks somehow. Not sure but possibly some regression in PyTorch.